packages/mui-styled-engine/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/styled-engine",
-  "version": "5.11.0",
+  "version": "5.11.8",
```

Comment:
@mnajdova Is this correct? I saw in the last release some patch numbers were aligned like this (went from `5.11.2` to `5.11.7`)

Reply:
Yes, this is correct.